network-policies/readme.adoc

```diff
@@ -0,0 +1,27 @@
+= Defining a Network Policy
```

Comment:
`cni` would be a better choice as mentioned at https://github.com/aws-samples/aws-workshop-for-kubernetes/pull/255#issuecomment-347362362

Reply:
Disagree, see https://github.com/aws-samples/aws-workshop-for-kubernetes/pull/255#issuecomment-347578679